README.md

```diff
@@ -30,236 +33,171 @@
-  and [DefaultShardManagerBuilder](https://docs.jda.wiki/net/dv8tion/jda/api/sharding/DefaultShardManagerBuilder.html)
+## Getting Started
 
-You can configure the memory usage by changing enabled `CacheFlags` on the `JDABuilder`.
-Additionally, you can change the handling of the member/user cache by disabling **intents** or changing the **member cache policy**.
-To learn more about intents and member loading/caching, read the [Gateway Intents Guide](https://jda.wiki/using-jda/gateway-intents-and-member-cache-policy/).
+We provide a number of [examples](https://github.com/discord-jda/JDA/tree/master/src/examples/java) to introduce you to JDA. You can also take a look at our official [Wiki][wiki], [Documentation][docs], and [FAQ][faq].
 
-```java
-public void configureMemoryUsage(JDABuilder builder) {
-    // Disable cache for member activities (streaming/games/spotify)
-    builder.disableCache(CacheFlag.ACTIVITY);
+Every bot implemented by JDA starts out using the [JDABuilder][JDABuilder] or [DefaultShardManagerBuilder][DefaultShardManagerBuilder]. Both builders provide a set of default presets for cache usage and events it wants to receive:
+
+- `createDefault` - Enables cache for users who are active in voice channels and all cache flags
+- `createLight` - Disables all user cache and cache flags
+- `create` - Enables member chunking, caches all users, and enables all cache flags
+
+We receommend reading the guide on [caching and intents](https://jda.wiki/using-jda/gateway-intents-and-member-cache-policy/) to get a feel for configuring your bot properly. Here are some possible use-cases:
 
-    // Only cache members who are either in a voice channel or owner of the guild
-    builder.setMemberCachePolicy(MemberCachePolicy.VOICE.or(MemberCachePolicy.OWNER));
+### Example: Message Logging
 
-    // Disable member chunking on startup
-    builder.setChunkingFilter(ChunkingFilter.NONE);
+A small-scale bot that simply logs messages to the console using the [JDABuilder][JDABuilder], intended for smaller bots that don't intend to grow to thousands of guilds
 
-    // Disable presence updates and typing events
-    builder.disableIntents(GatewayIntent.GUILD_PRESENCE, GatewayIntent.GUILD_MESSAGE_TYPING);
+Starting your bot and attaching an event listener, using the right [intents][GatewayIntent]:
 
-    // Consider guilds with more than 50 members as "large". 
-    // Large guilds will only provide online members in their setup and thus reduce bandwidth if chunking is disabled.
-    builder.setLargeThreshold(50);
+```java
+public static void main(String[] args) {
+  JDABuilder.createLight(token, EnumSet.of(GatewayIntent.GUILD_MESSAGES, GatewayIntent.MESSAGE_CONTENT))
+      .addEventListener(new MessageReceiveListener())
+      .build();
 }
 ```
 
-### Listening for Events
+**Important Note:** This also makes use of a **privileged** intent, which you have to explicitly enable in your application dashboard when you create your bot.
```

Comment:
I feel like using GitHub's Callout blocks is a good idea here for better visibility.
I.e.
```
> [!IMPORTANT]
> This also makes use of a **privileged** intent, which you have to explicitly enable in your application dashboard when you create your bot.
```
Result:
> [!IMPORTANT]
> This also makes use of a **privileged** intent, which you have to explicitly enable in your application dashboard when you create your bot.

Reply:
I didn't know this existed, that's very useful indeed.